packages/gp2-components/src/templates/BasicLayout.tsx

```diff
@@ -10,15 +10,36 @@ import { css } from '@emotion/react';
 import HeaderLogo from '../molecules/HeaderLogo';
 import Theme from './Theme';
 
-const { rem } = pixels;
+const { rem, vminLinearCalcClamped, mobileScreen, tabletScreen } = pixels;
 
 const contentStyles = css({
   borderTop: `1px solid ${steel.rgb}`,
   display: 'flex',
   flexDirection: 'column',
   alignItems: 'center',
   overflowY: 'auto',
+  width: '748px',
+  padding: `${vminLinearCalcClamped(
+    mobileScreen,
+    33,
+    tabletScreen,
+    48,
+    'px',
+  )} 0`,
+  margin: `0 auto`,
+  [drawerQuery]: {
+    maxWidth: '748px',
```

Comment:
Seems weird to set width and then max width to the same thing?

Reply:
the width's are set differently for different media-queries